### ISSUE TITLE
issues when enabling Opencv

### ISSUE THREAD

**Issue by Abbyls** [pjreddie/darknet#667]
Hi,
I encountered a probelm with "make" when I enabled the opencv.

the Makefile is :
GPU=0
CUDNN=0
OPENCV=1
OPENMP=0
DEBUG=0
...
when I make it, the error is:
" -DOPENCV -shared obj/gemm.o obj/utils.o obj/cuda.o obj/deconvolutional_layer.o obj/convolutional_layer.o obj/list.o obj/image.o obj/activations.o obj/im2col.o obj/col2im.o obj/blas.o obj/crop_layer.o obj/dropout_layer.o obj/maxpool_layer.o obj/softmax_layer.o obj/data.o obj/matrix.o obj/network.o obj/connected_layer.o obj/cost_layer.o obj/parser.o obj/option_list.o obj/detection_layer.o obj/route_layer.o obj/upsample_layer.o obj/box.o obj/normalization_layer.o obj/avgpool_layer.o obj/layer.o obj/local_layer.o obj/shortcut_layer.o obj/logistic_layer.o obj/activation_layer.o obj/rnn_layer.o obj/gru_layer.o obj/crnn_layer.o obj/demo.o obj/batchnorm_layer.o obj/region_layer.o obj/reorg_layer.o obj/tree.o obj/lstm_layer.o obj/l2norm_layer.o obj/yolo_layer.o -o libdarknet.so -lm -pthread  `pkg-config --libs opencv` 
/usr/bin/ld: cannot find -lcufft
/usr/bin/ld: cannot find -lcublas
/usr/bin/ld: cannot find -lnpps
/usr/bin/ld: cannot find -lnppi
/usr/bin/ld: cannot find -lnppc
/usr/bin/ld: cannot find -lcudart
collect2: error: ld returned 1 exit status
make: *** [libdarknet.so] Error 1"

I am confused why the error is related to cuda? And these lib(libcufft, libcublas, libnpps, libnppi, libnppc, libcudart) are existed in the path "/usr/local/cuda-8.0/targets/x86_64-linux/lib/".

Is there anyone know how to sove it?

**Comment by ah3243** [comment]
do you have the C++ api of opencv installed?
